src/codal_app/codal.json

```diff
@@ -10,6 +10,7 @@
         "DEVICE_STACK_SIZE": 8192,
         "MICROBIT_BLE_ENABLED" : 0,
         "MICROBIT_BLE_PAIRING_MODE": 1,
+        "MICROBIT_BLE_UTILITY_SERVICE_PAIRING": 1,
```

Comment:
Should this go in alphabetical order, ie after `MICROBIT_BLE_SECURITY_MODE`?

Reply:
Yeah, I placed it there to be next to the other pairing mode config option, but make sense to move it alphabetically 👍